CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2145, Fix accessing json array fields with -> and ->> in ?select= and ?order=. - @wolfgangwalther
  - #2153, Fix --dump-schema running with a wrong PG version. - @wolfgangwalther
  - #2042, Keep working when EMFILE(Too many open files) is reached. - @steve-chavez
+ - #2147, Using GET with `text/plain` or `application/octet-stream` as `Content-Type` headers no longer returns "Not Found" when the requested function with no parameters exists
```

Comment:
I had to read it a few times to understand.

What about the following?

```suggestion
 - #2147, Ignore `Content-Type` headers for `GET` requests when calling RPCs. Previously, `GET` without parameters, but with `Content-Type: text/plain` or `Content-Type: application/octet-stream` would fail with `404 Not Found`, even if a function without arguments was available.
 - ```

Reply:
> somehow make the Content-Type value part of the [Payload](https://github.com/PostgREST/postgrest/blob/main/src/PostgREST/Request/ApiRequest.hs#L71) type so it doesn't leak on GET logic?

Yeah, I think that's the right direction. We need stronger typing for our whole `ApiRequest`, eliminating many impossible cases.